internal/resolution/variablesources/bundle.go

```diff
@@ -94,7 +95,7 @@ func MakeBundleVariables(
 		// get bundle dependencies
 		dependencies, err := filterBundleDependencies(allBundles, head)
 		if err != nil {
-			return nil, fmt.Errorf("could not determine dependencies for bundle with id %q: %w", id, err)
+			return nil, fmt.Errorf("could not determine dependencies for bundle %q from package %q from catalog %q: %s", head.Name, head.Package, head.CatalogName, err)
```

Comment:
I don't like the wording: doesn't feel natural. Any suggestions?

Reply:
A couple options I could think of:
- `could not determine dependencies for bundle %q from package %q in catalog %q`. This is only a slight variation but prevents the repetition of the word "from"
- Something that includes additional context after the error message like: `could not determine dependencies for bundle %q. catalog: %q, package: %q`

Reply:
Simply replacing "from" with "in" sounds much better, thanks! I updated the PR with this version.